### ISSUE TITLE
[Research] OUI Compliance audit of the `management` plugin

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4085]
Tracking issue for OUI compliance audit of the `management` plugin.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

<details>
  <summary>Related files</summary>

  size (bytes) | file | notes
---|---|---
230 | [src/plugins/management/public/components/management_sidebar_nav/management_sidebar_nav.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/management/public/components/management_sidebar_nav/management_sidebar_nav.scss) | 
152 | [src/plugins/management/public/components/management_app/management_app.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/management/public/components/management_app/management_app.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
